contracts/EasyAuction.sol

```diff
@@ -156,7 +158,8 @@ contract EasyAuction is Ownable {
         uint256 minimumBiddingAmountPerOrder,
         uint256 minFundingThreshold,
         bool isAtomicClosureAllowed,
-        address allowListManager
+        address accessManagerContract,
+        bytes memory accessManagerContractData
```

Comment:
I think making this function external and making this field `calldata` might actually yield some gas savings.

Reply:
This would for some reason tricker one additional stack variable. Not sure what happens under the hood.
But I get with your proposed change: 
```
CompilerError: Stack too deep
```